src/dct/enum.lua

```diff
@@ -40,6 +40,8 @@ enum.assetType = {
 	["SPECIALFORCES"] = 22,
 	["FOB"]           = 23,
 	["SQUADRONPLAYER"]= 24,
+	["ARMEDRECCE"]    = 26,
```

Comment:
I still do not understand why this is needed. The change looks like this is just adding an asset type of the same name as the mission type, why? Considering the whole of the change the asset type "SPECIALFORCES" no longer appears for any mission type, neither does the asset type "FOB". Further the asset type "LOGISTICS" already appears under the BAI mission type. So why is any change other than adding the LOGISTICS type to the strategic class list needed?

I only see a need for a one line change at most, why are the other changes needed?

Reply:
Also as the below test run shows this change currently breaks the unit tests.

Reply:
Going to revisit this this week. The reason I made this is because we need the appropriate asset types to split out between ArmedRecce and BAI properly. E.G. A small camp somewhere would definitely be an Armed Recon target, whereas a COLUMN, which I have added here, would be a BAI target. It's just about trying to have granularity. Let me have a revisit of appropriate Asset types and their mission assignment.

I don't know why the unit test would be broken by the way, they're very small changes.

Reply:
We need to move towards asset types that have functional meaning for the AI commander. Since this doesn't currently exist, there is no reason to rename for example SpecialForces to ArmedRecce... it is just a definition in an enumeration at this point.

Additionally: the new asset types present in a branch or fork shouldn't be included in the master of another repo, such as the OEO templates repo. Instead that should be branched along with the code until the PRs are merged. As it stands now anyone attempting to use the OEO templates with a DCT release has to remove templates from multiple locations before the mission will work without errors. Then branches can be merged together. It also will allow us to better collect testing materials in one fell swoop if they are all named appropriately.